template/CordovaApp.Windows10.jsproj

```diff
@@ -86,12 +93,11 @@
     <Content Include="WinJS\js\WinJS.js" />
     <None Include="WinJS\js\WinJS.intellisense.js" />
     <None Include="WinJS\js\WinJS.intellisense-setup.js" />
-    <None Include="CordovaApp_TemporaryKey.pfx" />
   </ItemGroup>
-  <Import Project="CordovaApp.projitems" Label="Shared" />
+  <Import Project="CordovaApp.projitems" />
```

Comment:
I dont think we actually use this label but added back to make sure we dont break anything